test/src/test/java/hudson/tasks/BuildTriggerTest.java

```diff
@@ -309,13 +310,6 @@ public void downstreamProjectShouldObserveCompletedParent() throws Exception {
 
         final FreeStyleProject us = j.createFreeStyleProject();
         us.getPublishersList().add(new BuildTrigger("downstream", true));
-        us.getPublishersList().add(new TestNotifier() {
-            @Override
-            public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) throws InterruptedException, IOException {
-                Thread.sleep(5000); // Build does not complete just after the BuildTrigger
-                return true;
-            }
-        });
```

Comment:
I checked that the reproducer (021c1b3) for JENKINS-20989 still works even without this notifier. Therefore, it can be safely removed without making that test unable to catch future regressions (such as reverting the fix for JENKINS-20989).